client/components/domains/connect-domain-step/connect-domain-step-suggested-start.jsx

```diff
@@ -48,6 +47,35 @@ export default function ConnectDomainStepSuggestedStart( {
 				<br />
 				{ __( 'It can take up to 72 hours for the domain to be fully connected.' ) }
 			</p>
+			<FoldableCard
+				clickableHeader
+				className={ className + '__connected-services-card' }
+				header={
+					<div>
+						<Icon icon={ info } size={ 24 } />
+						{ __( 'Any services connected to this domain?' ) }
+					</div>
+				}
+				expanded={ false }
+			>
+				<p>
+					{ createInterpolateElement(
+						__(
+							'If you have any email or services other than web hosting connected to this domain, we recommend you copy over your DNS records before proceeding with this setup to avoid distruptions. You can then start the setup again by going back to <em>Upgrades > Domains</em>.'
+						),
+						{
+							em: createElement( 'em' ),
+						}
+					) }
+				</p>
+				<Button onClick={ goToDnsRecordsPage }>{ __( 'Go to DNS records' ) }</Button>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Back to DNS records' )` *ES Score: 7*